exercises/allergies/example.js

```diff
@@ -15,6 +15,7 @@ class Allergies {
   }
 
   list() {
+    // eslint-disable-next-line no-bitwise, no-restricted-properties
     return possibleAllergies.filter((allergy, i) => this.allergenIndex & Math.pow(2, i));
```

Comment:
That `&` should actually be replaced by `&&` - this will remove fix the `no-bitwise` error.

Reply:
Is that what we want here?  I think we actually want the bitwise operator, don't we?